src/Units/AWBWUnits.java

```diff
@@ -0,0 +1,778 @@
+package Units;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Vector;
+import Engine.GameAction;
+import Engine.UnitActionType;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.ResupplyEvent;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import Units.MoveTypes.Flight;
+import Units.MoveTypes.FloatHeavy;
+import Units.MoveTypes.FloatLight;
+import Units.MoveTypes.FootMech;
+import Units.MoveTypes.FootStandard;
+import Units.MoveTypes.MoveType;
+import Units.MoveTypes.Tires;
+import Units.MoveTypes.Tread;
+
+public class AWBWUnits extends UnitModelScheme
+{
+  private static final long serialVersionUID = 1L;
+
+  @Override
+  public String toString()
+  {
+    return "AWBW" + (schemeValid ? "" : "!");
```

Comment:
I might prepend the '!' so it looks less like punctuation. As an added bonus, '!' means "not" when prepended in most programming languages.

Reply:
I should probably standardize between the save file and this, too.
Do we want `~` or `!`, and should they be prepended or appended?

Reply:
I like having it up front, since that is a stronger demand for attention. Either works, but maybe '!', for the same reason.